client/components/sidebar.vue

```diff
@@ -171,7 +299,8 @@
         });
 
         const heights = this.$refs.comments.map((ref) => {
-          return ref.$el.offsetHeight;
+          // comment and replies container height (without comment input container) + comment card paddings
```

Comment:
Full sentence with capital first letter and dot at the end, for comments please.